code/modules/mob/living/simple_animal/borer/borer_html.dm

```diff
@@ -0,0 +1,129 @@
+
+/mob/living/simple_animal/borer/proc/get_html_template(content)
+	var/html = {"<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 4.01 Transitional//EN\" \"http://www.w3.org/TR/html4/loose.dtd\">
+		<html>
```

Comment:
This HTML code is a carbon copy from our codebase, back when we introduced these things before the rebase (meaning no TGUI), so that's why it's there. Plus, just like seen in the Cargo Share Computer Thing, it's much easier to define this inside of code, instead of making a separate file/css/adjustments.